README.md

```diff
@@ -191,6 +191,62 @@ The `items` key must point to a method that returns one of the following:
 - An array or collection of `FeedItem`s
 - An array or collection of arrays containing feed item values
 
+### Adding attachments
+
+You can add attachments to the feed like so.
+```php
+// app/NewsItem.php
+
+use Illuminate\Database\Eloquent\Model;
+use Spatie\Feed\Feedable;
+use Spatie\Feed\FeedItem;
+
+class NewsItem extends Model implements Feedable
+{
+    public function toFeedItem()
+    {
+        return FeedItem::create()
+            ->id($this->id)
+            ->title($this->title)
+            ->summary($this->summary)
+            ->updated($this->updated_at)
+            ->link($this->link)
+            ->author($this->author)
+            ->attachmentUrl($this->attachment_url)
+            ->attachmentSize($this->attachment_size)
+            ->attachmentType($this->attachment_type)
```

Comment:
I don't see this method in the `FeedItem` class. Am I missing something or some PHP magic?